### PR TITLE
chore: update packages and rename URL to GitHub repo

### PR DESCRIPTION
**Pull request by kelvinlouis**
Updating packages and changing URL to repository to GitHub

#GUILD-161

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://bitbucket.org/yooapps/yoo-digital-typescript.git"
+    "url": "https://github.com/yoo-digital/typescript.git"
```
Comment (RadLikeWhoa) [inline package.json]: Why not use ssh URLs like in the css package?

File: package.json [hunk 2]
```diff
@@ -64,8 +64,8 @@
     }
   ],
   "engines": {
-    "node": ">= 10",
-    "npm": ">= 6.14.0"
+    "node": ">= 16",
+    "npm": ">= 8.15.0"
```
Comment (RadLikeWhoa) [inline package.json]: Any reason for the more specific version constraint (compared to the css package)?

File: package.json (hunk 1 above)
Comment (kelvinlouis) [reply package.json]: Good catch, adjusted!

File: package.json (hunk 2 above)
Comment (kelvinlouis) [reply package.json]: No, there wasn't a reason. Adjusted to be consistent.

**Review by RadLikeWhoa — APPROVED** [review]

**Pull request merged**